LOGGING/src/EtwCollector/EtwCollector.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$(PropsRoot)\Product.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net452</TargetFrameworks>
+    <TargetFrameworks>net452;net462</TargetFrameworks>
```

Comment:
Adding `net462` might force us to support this project `Microsoft.ApplicationInsights.EtwCollector` for a long time. Should we reconsider adding net462 to this project?

Reply:
I agree with you, but I was reluctant to make that decision.